api/migrations/versions/c47b250bf785_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
Why are there two migration files for this change?

Reply:
Brought this up in slack yesterday afternoon.